### PR TITLE
Network Global Configuration moved to middlewared

### PR DESCRIPTION
**Pull request by sonicaj**
#Ticket: 30031

### CODE REVIEW

File: src/middlewared/middlewared/plugins/network.py [hunk 1]
```diff
@@ -17,6 +18,210 @@
...
+
+        await self.middleware.call(
+            'datastore.update',
+            'network.globalconfiguration',
+            config['id'],
+            new_config,
+            {'prefix': 'gc_'}
+        )
+
+        new_config['domains'] = new_config['netwait_ip'].split()
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/network.py]: Typo, new_config['netwait_ip']

**Comment by william-gr** [comment]
@themylogin can you review as well once you have time? thanks!

File: gui/network/forms.py [hunk 2]
```diff
@@ -588,153 +594,29 @@ def __init__(self, *args, **kwargs):
...
-            svcobj = services.objects.get(srv_service='nfs')
-            nfsobj = NFS.objects.all()[0]
-            if (svcobj and svcobj.srv_enable) and (nfsobj and (nfsobj.nfs_srv_v4 and nfsobj.nfs_srv_v4_krb)):
-                notifier().restart("ix-nfsd")
-                whattoreload.append("mountd")
-
-        for what in whattoreload:
-            notifier().reload(what)
+    def middleware_clean(self, update):
+        update['domains'] = update['domains'].split()
```
Comment (themylogin) [inline gui/network/forms.py]: `list(filter(None, re.split(r"\s+", update["domains"])))` here and everywhere else would work better

**Comment by sonicaj** [comment]
rebuild

**Pull request merged**